### ISSUE TITLE
Error running command pr.pick: Cannot read property 'startsWith' of undefined

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode-pull-request-github#1414]
![image](https://user-images.githubusercontent.com/2193314/67564631-3b505500-f6d8-11e9-806b-ea3843e2ec7d.png)

This happened when checking out this vscode PR:

![image](https://user-images.githubusercontent.com/2193314/67564664-515e1580-f6d8-11e9-97f7-64162ed6fea7.png)

It worked the second time.

Output channel:

<details><pre>
[Info] Looking for git repository
[Info] Git repository found, initializing review manager and pr tree view.
[Info] Review> Validate state in progress
[Info] Review> Validating state...
[Info] No remote with name 'origin' found. Please update your 'githubPullRequests.remotes' setting.
[Info] No remote with name 'ups' found. Please update your 'githubPullRequests.remotes' setting.
[Info] No GitHub remotes found
[Info] Review> Queuing additional validate state
[Info] Review> Validating state...
[Info] Review> Queuing additional validate state
[Info] Review> Validating state...
[Info] Review> Queuing additional validate state
[Info] Review> Queuing additional validate state
[Info] Review> current branch master is associated with pull request #83175
[Info] Review> Resolving pull request
[Info] Review> This PR is no longer valid
[Info] Review> Validating state...
[Info] No remote with name 'ups' found. Please update your 'githubPullRequests.remotes' setting.
[Info] Found GitHub remote
[Info] https://api.github.com: GraphQL support detected
[Info] Review> no matching pull request metadata found for current branch master
[Info] Review> Validating state...
[Info] No remote with name 'ups' found. Please update your 'githubPullRequests.remotes' setting.
[Info] Found GitHub remote
[Info] Review> no matching pull request metadata found for current branch master
[Info] Review> Queuing additional validate state
[Info] Review> Validating state...
[Info] No remote with name 'ups' found. Please update your 'githubPullRequests.remotes' setting.
[Info] Found GitHub remote
[Info] Review> no matching pull request metadata found for current branch master
[Info] Review> Queuing additional validate state
[Info] Review> Validating state...
[Info] No remote with name 'ups' found. Please update your 'githubPullRequests.remotes' setting.
[Info] Found GitHub remote
[Info] Review> no matching pull request metadata found for current branch master
[Info] Review> switch to Pull Request #83175 - start
[Info] Review> Review> switch to Pull Request #83175 - done
[Info] Review> Queuing additional validate state
[Info] Review> Validating state...
[Info] No remote with name 'ups' found. Please update your 'githubPullRequests.remotes' setting.
[Info] Found GitHub remote
[Info] Review> current branch pr/jmbockhorst/83175 is associated with pull request #83175
[Info] Review> Resolving pull request
[Info] Review> Fetching pull request data
[Info] Review> register comments provider
[Info] Review> display pull request status bar indicator and refresh pull request tree view.
</pre></details>

**Comment by alexr00** [comment]
Closing as a one off error. If you see it again please comment!

**Comment by lucasrodes** [comment]
Hi @alexr00,
This is raising an error for me when trying to check out to a PR.

My VS Code:
```
Version: 1.63.2
Commit: 899d46d82c4c95423fb7e10e68eba52050e30ba3
Date: 2021-12-15T09:37:28.172Z
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Darwin x64 20.6.0
```

Running on Arch Linux, connected remotely.

```
~ uname -a
Linux user 5.10.84-1-MANJARO #1 SMP PREEMPT Wed Dec 8 09:50:30 UTC 2021 x86_64 GNU/Linux
```


The developer tools are showing the following after clicking on "checkout to PR":

![image](https://user-images.githubusercontent.com/18101289/147750392-721e5121-3f20-4bd2-9ad8-8cb8d283280c.png)

I appreciate any help

**Comment by alexr00** [comment]
@lucasrodes is there more of a stack trace in the dev tools?

I've made some guesses on places where `startsWith` could be called on undefined and pushed a fix. If you could please try out the next [pre-release build](https://github.com/microsoft/vscode-pull-request-github/wiki/Pre-release-Build) (should be available on Monday with version number 0.35.20220103*), that would help determine if the issue is fixed.

**Comment by lucasrodes** [comment]
Hi @alexr00,
Thanks for your rapid response. It seems that this error has now disappeared. However, I am still encountering some other issues. I created the separate issue #3217 where I explain it more in detail.

Thanks :)
